src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $main-card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+    cursor: pointer;
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+
+    width: 160px;
+    height: 134px;
+  }
```

Comment:
Consider using a mixin for setting the width and height for images to make the code more maintainable and flexible. This would allow you to reuse the same styles with different values if needed.